guides/v2.2/cloud/before/before-workspace-cloud-account.md

```diff
@@ -21,5 +18,5 @@ We recommend always starting with the blank site from a template as your initial
 
 {% include cloud/new-project-from-template.md %}
 
-#### Next step
+**Next step**
```

Comment:
```suggestion
**Next step:**
```

Reply:
I see the inconsistency here. Ugh...inherited inconsistency...ggrrrrrrr   <pulling hair out>